app/assets/javascripts/application.js

```diff
@@ -1 +1,18 @@
 //= require 'uswds/uswds'
+//= require 'jquery'
+//= require 'owl-carousel2'
+
+(function(window, $) {
+  'use strict';
+
+  $('.owl-carousel').owlCarousel({
```

Comment:
> Where does rails put this JS file?

Rails' [Asset Pipeline](http://guides.rubyonrails.org/asset_pipeline.html) will look in a couple of places for assets (see [section 2.2](http://guides.rubyonrails.org/asset_pipeline.html#asset-organization)):

- `app/assets/javascripts`
- `lib/assets/javascripts`
- `vendor/assets/javascript`
- …and then (I think) in any gems that provide assets.

> does this need a $(document).ready()?

Good question. _Maybe?_ The `application.js` is included [at the bottom of `application.html.erb`](https://github.com/deptofdefense/move.mil/blob/master/app/views/layouts/application.html.erb#L155-L164) so the JS _should_ execute pretty late in the game.